pkg/auth/handler/webapp/authflow_forgot_password_success.go

```diff
@@ -0,0 +1,52 @@
+package webapp
+
+import (
+	"net/http"
+
+	"github.com/authgear/authgear-server/pkg/auth/handler/webapp/viewmodels"
+	"github.com/authgear/authgear-server/pkg/auth/webapp"
+	"github.com/authgear/authgear-server/pkg/util/httproute"
+	"github.com/authgear/authgear-server/pkg/util/template"
+)
+
+var TemplateWebAuthflowForgotPasswordSuccessHTML = template.RegisterHTML(
+	"web/authflow_forgot_password_success.html",
+	components...,
+)
+
+func ConfigureAuthflowForgotPasswordSuccessRoute(route httproute.Route) httproute.Route {
+	return route.
+		WithMethods("OPTIONS", "GET").
+		WithPathPattern(webapp.AuthflowRouteForgotPasswordSuccess)
+}
+
+type AuthflowForgotPasswordSuccessHandler struct {
+	Controller    *AuthflowController
+	BaseViewModel *viewmodels.BaseViewModeler
+	Renderer      Renderer
+}
+
+func (h *AuthflowForgotPasswordSuccessHandler) GetData(w http.ResponseWriter, r *http.Request, s *webapp.Session, screen *webapp.AuthflowScreenWithFlowResponse) (map[string]interface{}, error) {
+	data := make(map[string]interface{})
+
+	baseViewModel := h.BaseViewModel.ViewModel(r, w)
+	viewmodels.Embed(data, baseViewModel)
+
+	return data, nil
+}
+
+func (h *AuthflowForgotPasswordSuccessHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	var handlers AuthflowControllerHandlers
```

Comment:
We do not actually need data from the screen. Maybe we can skip using AuthflowController here.

Reply:
Using `Controller.HandleStep` here will make sure the user see this page only if his account recovery flow is on the correct step.